backend/routes.py

```
@@ -1314,16 +1344,46 @@
+                return jsonify({'error': 'User management privileges required'}), 403
+
+            # Store user info in g for use in the route
+            g.current_user = current_user
+            g.current_user_id = current_user.id
+            g.is_admin = current_user.is_admin
+
+        except jwt.ExpiredSignatureError:
+            return jsonify({'error': 'Authentication required', 'reason': 'Token expired'}), 401
+        except jwt.InvalidTokenError:
+            return jsonify({'error': 'Authentication required', 'reason': 'Invalid token'}), 401
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)